nautobot/core/api/serializers.py

```diff
@@ -54,10 +58,10 @@ def fields(self):
 
             # NOTE: drf test framework builds a request object where the query
             # parameters are found under the GET attribute.
-            params = getattr(request, "query_params", getattr(request, "GET", None))
+            params = normalize_querydict(getattr(request, "query_params", getattr(request, "GET", None)))
 
             try:
-                user_opt_in_fields = params.get("include", None).split(",")
+                user_opt_in_fields = params.get("include", [])
             except AttributeError:
                 # include parameter was not specified
                 user_opt_in_fields = []
```

Comment:
Do we still need to catch exceptions here?

Reply:
Probably not - but harmless to leave in place, I think?